kernels/flash-attn/flash_attn_mma.py

```diff
@@ -19,6 +19,14 @@
     precision=6, threshold=8, edgeitems=3, linewidth=120, sci_mode=False
 )
 
+CUTLASS_REPO_PATH = os.environ.get(
+    "CUTLASS_REPO_PATH", os.path.expanduser("~/cutlass")
+)
+
+CUTLASS_REPO_PATH = os.environ.get(
+    "CUTLASS_REPO_PATH", os.path.expanduser("~/cutlass")
+)
```

Comment:
The cutlass is already inside `-I {project_dir}/third-party/cutlass` as a submodule, please use the cutlass submodule in LeetCUDA/third_party, for example: 

https://github.com/xlite-dev/LeetCUDA/blob/a566b8814a9ece1b4345131c570f60433052f814/kernels/hgemm/tools/utils.py#L94

Reply:
> run:
> 
> ```shell
> cd LeetCUDA
> git submodule update --init --recursive --force
> ```

@DefTruth removed in https://github.com/xlite-dev/LeetCUDA/pull/314/commits/d4aa3b246a81b9581d12bf07b66c404b5db97c2e